common/djangoapps/track/views/tests/test_segmentio.py

```diff
@@ -404,3 +405,119 @@ def test_video_event(self, name, event_type):
 
         self.assertEqualUnicode(actual_event, expected_event_without_payload)
         self.assertEqualUnicode(payload, expected_payload)
+
+    @data(
+        (1, 1, "seek_type", "slide", "onSlideSeek", "edx.video.seeked", "edx.video.position.changed", 'edx.mobileapp.iOS', '1.0.02'),
+        (-2, -2,"seek_type", "slide", "onSlideSeek", "edx.video.seeked", "edx.video.position.changed", 'edx.mobileapp.iOS', '1.0.02'),
+        (30, -30, "seek_type", "skip", "onSkipSeek", "edx.video.position.changed", "edx.video.position.changed", 'edx.mobileapp.iOS', '1.0.02'),
+        (-30, -30, "seek_type", "skip", "onSkipSeek", "edx.video.position.changed", "edx.video.position.changed", 'edx.mobileapp.iOS', '1.0.02'),
+        (1, 1, "type", "onSlideSeek", "onSlideSeek", "edx.video.seeked", "edx.video.position.changed", 'edx.mobileapp.android', '1.0.02'),
+        (-2, -2, "type", "onSlideSeek", "onSlideSeek", "edx.video.seeked", "edx.video.position.changed", 'edx.mobileapp.android', '1.0.02'),
+        (30, 30, "type", "onSkipSeek", "onSkipSeek", "edx.video.position.changed", "edx.video.position.changed",'edx.mobileapp.android', '1.0.02'),
+        (-30, -30, "type", "onSkipSeek", "onSkipSeek", "edx.video.position.changed", "edx.video.position.changed", 'edx.mobileapp.android', '1.0.02')
+    )
+    @unpack
+    def test_previous_builds(self,
+                             requested_skip_interval,
+                             expected_skip_interval,
+                             seek_type_key,
+                             seek_type,
+                             expected_seek_type,
+                             name,
+                             expected_name,
+                             platform,
+                             version
+    ):
+        """
+        Test backwards compatibility of previous app builds
+
+        iOS version 1.0.02 incorrect emits the skip back 30 seconds as +30
+        instead of -30.
+        Android version 1.0.02 Skip and slide were both being returned as a
+        skip. Skip or slide is determined by checking if the skip time is == -30
```

Comment:
Also, to add in the docstring: edx.video.seeked was sent as the seek name instead of edx.video.position.changed.